packages/lesswrong/lib/givingSeason/index.ts

```diff
@@ -1,4 +1 @@
-export const ACTIVE_DONATION_ELECTION = "givingSeason24";
-export const DONATION_ELECTION_NUM_WINNERS = 3;
-export const DONATION_ELECTION_SHOW_LEADERBOARD_CUTOFF = 100;
-export const DONATION_ELECTION_AGE_CUTOFF = new Date("2024-10-22");
+export const ACTIVE_DONATION_ELECTION = "givingSeason24"; // TODO bump to current year if we do another donation election
```

Comment:
I'd probably just delete this. The only place it's still used is in the election candidates schema, but it doesn't _actually_ effect the schema - it's just an option in a dropdown (which will never be chosen again).